github/ci/prow/files/jobs/kubevirt/project-infra/project-infra-presubmits.yaml

```diff
@@ -12,11 +12,11 @@ presubmits:
         args:
```

Comment:
This may become a problem, as configuation is dynamically generated, you can't just checkconfig anymore, you would need to generate and test them after.